gnome-initial-setup/pages/site/gis-site-page.c

```diff
@@ -0,0 +1,475 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*- */
+/*
+ * Copyright (C) 2012 Red Hat
+ * Copyright (C) 2018 Endless Technologies, Inc.
```

Comment:
I am not sure we ever formally renamed the company to this name. I'd use `Endless Mobile, Inc.`